app/src/main/java/com/sun/basic_japanese/alphabet/adapter/AlphabetRecyclerAdapter.kt

```diff
@@ -0,0 +1,49 @@
+package com.sun.basic_japanese.alphabet.adapter
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import com.sun.basic_japanese.R
+import com.sun.basic_japanese.constants.BasicJapaneseConstants.HIRAGANA
+import com.sun.basic_japanese.data.model.Alphabet
+
+class AlphabetRecyclerAdapter(
+    private val alphabetItems: MutableList<Alphabet?>,
+    private val alphabetType: String?
+) : RecyclerView.Adapter<AlphabetRecyclerAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, position: Int): ViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        val view = inflater.inflate(R.layout.item_word, parent, false)
+        return ViewHolder(view)
+    }
+
+    override fun onBindViewHolder(viewHolder: ViewHolder, position: Int) {
+        viewHolder.bindData(alphabetItems[position], alphabetType)
+    }
+
+    override fun getItemCount() = alphabetItems.size
+
+    class ViewHolder
+        (itemView: View) : RecyclerView.ViewHolder(itemView) {
+        private val textWordLabel: TextView? = itemView.findViewById<TextView>(R.id.textAlphabetJapanese)
+        private val textWordDescription: TextView? = itemView.findViewById<TextView>(R.id.textAlphabetInternational)
```

Comment:
```suggestion
     itemView.apply {
                textAlphabetJapanese?.text = alphabet.hiragana
            }
```
Có thể viết như thế này không cần findviewbyid